app/src/main/java/mozilla/lockbox/store/AccountStore.kt

```diff
@@ -38,9 +38,10 @@ import mozilla.lockbox.support.Constant
 import mozilla.lockbox.support.Optional
 import mozilla.lockbox.support.SecurePreferences
 import mozilla.lockbox.support.asOptional
+import java.lang.Exception
 import java.util.concurrent.TimeUnit
 import kotlin.coroutines.CoroutineContext
-import org.mozilla.fxaclient.internal.FxaException as FxaException
+import org.mozilla.fxaclient.internal.FxaException
```

Comment:
Is this still needed now?

Reply:
Yep! it's still used, it's just not complaining about the alias anymore